.github/workflows/python.yaml

```diff
@@ -36,7 +36,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        python-version: ['3.10']
+        python-version: ['3.8', '3.9', '3.10', '3.11', '3.12']
```

Comment:
Much of the scientific python community has already dropped Python 3.8 support per NEP 29:

https://numpy.org/neps/nep-0029-deprecation_policy.html

Up to you of course what to support, but maybe not worth going as far back as 3.8 (which is EOL this year for Python as well)